packages/mermaid/src/diagram-api/types.ts

```diff
@@ -37,6 +45,7 @@ export interface DiagramDB {
 
   setDisplayMode?: (title: string) => void;
   bindFunctions?: (element: Element) => void;
+  getMessages?: () => Message[];
```

Comment:
We cannot add this type to DiagramDB, as it's not a common function that each diagram shares. 
It'll have to be added to the specific diagramDB which exposes the getMessages function.

Reply:
This looks like will take more time as there are no types for sequenceDb so I will do that in a separate PR